src/main/java/rotateit/RotateItApplication.java

```diff
@@ -4,6 +4,7 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
+@SuppressWarnings("PMD")
```

Comment:
Classes annotated `@SuppressWarnings("PMD")` has few types of warning and should be fixed. I just haven't time for this. I will back to it.